src/main/java/net/datafaker/providers/healthcare/CareProvider.java

```diff
@@ -0,0 +1,28 @@
+package net.datafaker.providers.healthcare;
+
+import net.datafaker.providers.base.AbstractProvider;
+
+/**
+ * Generate random types of health care providers
```

Comment:
All providers require a "since" property, so I think a since 2.3.0 would be good, even though we might release this as part of 2.2.3, not sure yet.

Reply:
Got it. I added it in all new classes and interface.